bb-core/src/main/java/com/cognifide/qa/bb/api/actions/ActionResponse.java

```diff
@@ -0,0 +1,30 @@
+/*-
+ * #%L
+ * Bobcat
+ * %%
+ * Copyright (C) 2018 Cognifide Ltd.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.cognifide.qa.bb.api.actions;
+
+/**
+ * Generic interface for action response
+ * @param <T>
+ */
+public interface ActionResponse<T> {
```

Comment:
Do we actually need this...? Can't we parametric the `WithResponse` interface? Having this, will force people to write wrappers for various types, instead of simply using them.

Reply:
Done